app/src/main/res/values/ic_launcher_background.xml

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <color name="ic_launcher_background">@color/blue_175DDC</color>
-    <color name="ic_shortcut_background">@color/white</color>
+    <!-- This is the apps Primary Blue color an exists here for launcher and splash theme only -->
+    <color name="ic_launcher_background">#FF175DDC</color>
```

Comment:
This one is a little interesting, we need the color to be defined in XML for both the launcher icon and the splash theme. This is the only color in the app that needs to be defined in XML.